bundles/EcommerceFrameworkBundle/PaymentManager/Payment/PaymentProviderInterface.php

```diff
@@ -0,0 +1,33 @@
+<?php
```

Comment:
remove this as it is not needed anymore.

Reply:
Removed